src/PrestaShopBundle/Api/Api.php

```diff
@@ -33,7 +33,7 @@
  */
 final class Api
 {
-    public const API_BASE_PATH = '/new-api';
+    public const API_BASE_PATH = '/funkypapi';
```

Comment:
I'd simply use `/api` even if we already have existing APIs with this prefix I don't think it's a problem to use the same, the Swagger doc will only output the API platform endpoints anyway

Besides, even the token endpoint uses this prefix currently `/api/oauth/token` so why not uniformize this And the existing APIs will likely be replaced and removed by the new ones in time